scripts/cat/cats.py

```diff
@@ -2900,103 +2920,106 @@ def mediate_relationship(mediator, cat1, cat2, allow_romantic, sabotage=False):
             else:
                 bonus = 0
 
-            decrease: bool = sabotage
+            if sabotage or value in chosen_neg:
+                decrease = True
+            else:
+                decrease = False
```

Comment:
```suggestion
                decrease = sabotage or value in chosen_neg
```
(lines `2923-2926` inclusive, just in case it decides to only pick the bottom one)